activerecord/lib/active_record/associations.rb

```diff
@@ -1251,6 +1273,14 @@ module ClassMethods
         #   persisted new records placed at the end.
         #   When set to +:nested_attributes_order+, the index is based on the record order received by
         #   nested attributes setter, when accepts_nested_attributes_for is used.
+        # [:before_add]
+        #   Defines an association callback[rdoc-ref:Associations::ClassMethods@Association+callbacks] that gets triggered before an object is added to the association collection.
```

Comment:
Maybe add emphasis around the important part? Otherwise these just kind of blend in to me.

```suggestion
        #   Defines an association callback[rdoc-ref:Associations::ClassMethods@Association+callbacks] that gets triggered **before an object is added** to the association collection.
```

Double check the RDoc syntax for bold, I'm always confusing bold and italic.

I did check the links though, and it works. I'm not sure if we want to use `{association callback}[rdoc-ref:...]` since this will only link the single word before it, and another concern is about compatibility when this doc eventually gets converted to markdown. So maybe stick with the `{}[]` link form?

Reply:
According to the [RDoc Syntax guides](https://docs.ruby-lang.org/en/master/RDoc/MarkupReference.html#class-RDoc::MarkupReference-label-Bold), wrapping with a single `*` should bold.

Reply:
After a rebuild, I was able to check and it looks like that didn't work:
https://44927d65.rails-docs-preview.pages.dev/api/classes/ActiveRecord/Associations/ClassMethods

![Screenshot from 2024-09-12 10-43-15](https://github.com/user-attachments/assets/52023751-0603-4283-ac84-2a42bdd39867)

Reply:
@zzak I've replaced the `*` with `<b>`, and that worked locally.